cmd/tvx/main.go

```diff
@@ -5,9 +5,21 @@ import (
 	"os"
 	"sort"
 
+	"github.com/filecoin-project/go-jsonrpc"
 	"github.com/urfave/cli/v2"
+
+	"github.com/filecoin-project/lotus/api"
+	lcli "github.com/filecoin-project/lotus/cli"
 )
 
+// FullAPI is a JSON-RPC client targeting a full node. It's initialized in a
+// cli.BeforeFunc.
+var FullAPI api.FullNode
+
+// Closer is the closer for the JSON-RPC client, which must be called on
+// cli.AfterFunc.
+var Closer jsonrpc.ClientCloser
```

Comment:
These are now global vars in the main package initialized by the `initialize` `cli.BeforeFunc`.